packages/playwright-core/src/utils/httpServer.ts

```diff
@@ -128,7 +128,8 @@ export class HttpServer {
       this._urlPrefixHumanReadable = address;
     } else {
       this._port = address.port;
-      const resolvedHost = address.family === 'IPv4' ? address.address : `[${address.address}]`;
+      const isIPV4 = address.family === 'IPv4' || (address.family as any) === 4; // Node 18.0.0 returns number, >18.4.0 returns string
```

Comment:
I don't think we support 18.0.0. We support only the latest 18.X, 20.X, 22.X release.

I recommend the user instead update his Node.js since he misses important security updates